src/Arm.cpp

```diff
@@ -25,78 +26,227 @@ Arm::Arm(
+               (m_rightPotentiometer->GetVoltage() > m_configEditor->getFloat("minSoftLimitRight"))){
+            powerLeft = m_armMotorPower;
+            powerRight = m_armMotorPower;
          }
          //}
       }
-
+      m_armMotorLeft->Set(powerLeft);
+      m_armMotorRight->Set(powerRight);
    }
-   //   else{
-   //      if (fabs(pAngle - getAngleLeft()) < RobotConstants::armDeadZone){
-   //         //TODO add in limit switch logic
-   //         if(pAngle < getAngleLeft()){
-   //            if (!m_leftLowerLimitSwitch->Get() && !m_rightLowerLimitSwitch->Get()){
-   //               power = m_armMotorPower;
-   //            }
-   //            //power =-m_armMotorPower;
-   //         }
-   //         else if(pAngle > getAngleLeft()){
-   //            if (!m_leftUpperLimitSwitch->Get() && !m_rightUpperLimitSwitch->Get()){
-   //               power = m_armMotorPower;
-   //            }
-   //            //power= m_armMotorPower;
-   //         }
-   //      }
-   //   }
-   m_armMotorLeft->Set(power);
-   m_armMotorRight->Set(power);
 }
+   void Arm::manualDrive(){
 
-void Arm::move(float motorPower){
-   PDRIVE = false;
-   m_armMotorPower = motorPower;
-}
+      m_armMotorPower = m_driveStation->getGamepadJoystick() / 2;
 
-void Arm::stop(){
-   PDRIVE = false;
-   m_armMotorPower = 0;
-}
 
-void Arm::angle(float angle,float motorpower){
-   m_armMotorPower = motorpower;
-   pAngle = angle;
-   PDRIVE = true;
-}
+      //   else{
+      //      if (fabs(pAngle - getAngleLeft()) < RobotConstants::armDeadZone){
+      //         //TODO add in limit switch logic
+      //         if(pAngle < getAngleLeft()){
+      //            if (!m_leftLowerLimitSwitch->Get() && !m_rightLowerLimitSwitch->Get()){
+      //               power = m_armMotorPower;
+      //            }
+      //            //power =-m_armMotorPower;
+      //         }
+      //         else if(pAngle > getAngleLeft()){
+      //            if (!m_leftUpperLimitSwitch->Get() && !m_rightUpperLimitSwitch->Get()){
+      //               power = m_armMotorPower;
+      //            }
+      //            //power= m_armMotorPower;
+      //         }
+      //      }
+      //   }
+   }
 
-float Arm::getAngleLeft(){
-   double relativeVoltLeft = (m_leftPotentiometer->GetVoltage() - m_configEditor->getFloat("potLeftValueLow"))/(m_configEditor->getFloat("potLeftValueHigh") - m_configEditor->getFloat("potLeftValueLow"));
-   return relativeVoltLeft * (RobotConstants::maxTheta - RobotConstants::minTheta);
-}
+   void Arm::run(){
 
-float Arm::getAngleRight(){
-   double relativeVoltRight = (m_rightPotentiometer->GetVoltage() - m_configEditor->getFloat("potRightValueLow"))/(m_configEditor->getFloat("potRightValueHigh") - m_configEditor->getFloat("potRightValueLow"));
-   return relativeVoltRight* (RobotConstants::maxTheta - RobotConstants::minTheta);
-}
+
+      std::ostringstream leftPotentiometer;
```

Comment:
Please remove all 'SmartDashboard' prints from files other than robot.cpp and driverstation.cpp